src/js/utils/Browser.js

```diff
@@ -1,6 +1,51 @@
 // 'Inspired' by https://github.com/clappr/clappr/blob/master/src/components/browser.js
+import { canBrowserAutoplay, canBrowserAutoplayMuted } from './services/canBrowserAutoplay';
 
-const Browser = {};
+const Browser = {
+    /**
+     * Retrieves browser info in a asynchronous manner
+     * Please use this instead of accessing properties directly.
+     *
+     * @returns {Promise<object>}
+     */
+    async getInfo() {
```

Comment:
I'm thinking we should be caching this.

Reply:
Yes, since it's 22 nano seconds i think we for now can delay this.